### ISSUE TITLE
Recommended way to convert old Google VR Unity project to current version

### ISSUE THREAD

**Issue by lexar** [googlevr/gvr-unity-sdk#901]
Summary:
I have a Unity VR project based on Unity 5.4 and Google VR 1.0 without native integration (I think, enable VR in Player settings is disabled).
I would like to upgrade to LTS-Version 2017.4.1f1 with the Google VR included.
What is the recommended way to do that? Specifically, how do I handle the old Google VR 1.0 package in the current project? Do I just delete it? Thanks, lexar

**Comment by rusmaxham** [comment]
[This old README](https://github.com/googlevr/gvr-unity-sdk/tree/a3d1033260dab57cb0f4a62a770796fbd09fe37a) has migration steps for upgrading from pre 1.40 SDK to current.  You can use Unity 2017.4 instead of 5.6.

**Comment by lexar** [comment]
Thanks! But I'm still uncertain how to handle that old package. Should I simply import the appropriate unity package and let it overwrite anything?

**Comment by rusmaxham** [comment]
Yes.  The 1.40 SDK will delete some of the outdated plugin files when you open it in Unity 5.6 or later.  Then importing a new SDK on top of that should work.
